core/controllers/classroom.py

```diff
@@ -78,6 +79,7 @@ def get(self, classroom_name):
         ]
 
         self.values.update({
-            'topic_summary_dicts': topic_summary_dicts
+            'topicSummaryDicts': case_change_service.camelize(
```

Comment:
It's a good idea to have a general solution. However, the implementation needs to be planned out a bit more. Ultimately, what would be best is to have the camelization done in self.render_json on the overall dict, so that it applies to all existing and future handlers and that we have a uniform convention throughout. Otherwise you end up with some handlers being camelized and some not, and that is confusing because it messes up conventions.

So I would suggest that you either do the above for all handlers at a time, or for no handlers. This will be a little grueling, but it would be better than having discrepant conventions in different handlers in the codebase. I think it's fine if the PR is largish, if you keep the PR focused on only that change and don't bundle anything else into it.

Reply:
Hi @seanlip, I agree with you. We should camelize the complete response. I have made a systematic plan for it and included it in my [GSoC proposal](https://docs.google.com/document/d/1fIoXVtO3Br2BtuR2B8XnsmJsHTAd3SjsGhUbjENJuPc/edit#bookmark=id.v9573m8ir9lf). Please give it a look and tell if it seems fine to you.

Reply:
No, I don't think so. The net intermediate result of the plan you've proposed is a bunch of hybrid responses where some fields in the response are camelized and some are not. This will create confusion, so please don't do it.

As mentioned above, I think you need a single PR that changes everything in one go, in order to avoid developer confusion.

Reply:
Also, see my note above re using self.render_json for this, not python_utils. This isn't a python-2-to-3 thing.

Reply:
Hi @seanlip, I think you misunderstood me a bit. In my proposal, I described my approach to camelize the responses systematically. Ultimately I will make just **one PR**, camelizing the complete response of all the endpoints. I think it's unclear in the proposal, I will make it more clear. 

And for the `python_utils` thing, So, should I write the camelize function in the `BaseHandler` itself?

Reply:
For the first part: the proposal currently does appear to describe something different, so clarifying it would be good. If you are talking about your "internal approach", that is reasonable to document, but is secondary, so perhaps delineate it somehow.

For the second part: yes.

Reply:
@seanlip Ok, So I think I should close this PR for now.